### DISCUSSION TITLE
Request for input for v1.x release

### DISCUSSION THREAD

**Discussion by guerzon** [guerzon/vaultwarden#64] [Ideas]
Hello,

We're gathering suggestions from Kubernetes and Helm users on what else they would like to be added to the chart before marking it as v1.x version (i.e. "can be used in production").

Please feel free to write your suggestions and requests in this discussion.

Thanks!

**Comment by hagak** [comment]
Just getting started with this chart, however it does not appear to be able to use a pre-existing PVC for the data storage.  I use volsync to backup my PVC and as such have templates to make this easy and consistent between my applications.  Could you add the ability to specify an "existingClaim" under the data key?
